vector/api/endpoints/fetch-random.mdx

```diff
@@ -23,8 +23,11 @@ The response will be `null` if the namespace is empty.
 <ResponseField name="id" type="string" required>
   The id of the vector.
 </ResponseField>
-<ResponseField name="vector" type="number[]" required>
-  The vector value.
+<ResponseField name="vector" type="number[]">
+  The dense vector value for dense and hybrid indexes.
+</ResponseField>
+<ResponseField name="sparseVector" type="Object[]">
```

Comment:
```suggestion
<ResponseField name="sparseVector" type="number[]">
```

Reply:
No, this is an array of something like `[{"indices": [...], "values": [...]}, ...]`